tehuti.py

```diff
@@ -168,6 +170,45 @@ def run(self):
         return [sum(vals) / self.number for vals in self.usage_log]
 
 
+class AccuracyMetric(Metric):
+    class Context(object):
+        pass
+
+    def __init__(self, body, setup=None, repeat=100, number=1, name=None):
```

Comment:
I guess I was thinking in terms of regridders when I wrote this, and made the assertion that I wasn't sure that the regridder would always return the same values, so allowed the metric to capture such variability.